luavm/Cargo.toml

```diff
@@ -8,7 +8,10 @@ name = "luavm"
 path = "src/lib/mod.rs"
 
 [dependencies]
+gc = "*"
```

Comment:
Shouldn't we be specifying `0.3` or similar?

Reply:
Missed that!